packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Connect your SCIM API service to Okta
+excerpt: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+meta:
+  - name: description
+    content: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+layout: Guides
+---
+
+This guide — part of the [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview/) series — teaches you how to create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+
+## Getting connected
+
+After you have a SCIM implementation that passes all of the Runscope tests, as discussed in the [previous part of the series](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api), you need to create your SCIM integration directly within Okta.
```

Comment:
How about replacing this paragraph (NOTE, I didn't know the title of the article... "previous part of the series" could be confusing if the developer gets here directly from a search) :
Make sure your SCIM implementation passes all the Runscope tests before integrating it with Okta. For more information on resting your SCIM implementation, see [TITLE OF THE ARTICLE GOES HERE](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api)
Before integration your SCIM using Okta

Reply:
done